src/components/Menu/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu'
+import { socialLinks } from './utils/socialLinks'
+
+interface DropdownProps {
+  children: React.ReactNode
+}
+
+export function Dropdown({
+  children
+}: DropdownProps) {
+  return (
+    <DropdownMenu.Root>
+      <DropdownMenu.Trigger asChild>
+        {children}
+      </DropdownMenu.Trigger>
+
+      <DropdownMenu.Portal>
+        <DropdownMenu.Content side='right' className="bg-[#2a273f] z-10 flex flex-col gap-2 rounded-lg overflow-auto">
+          {socialLinks.map(socialItem => {
+            return (
+              <DropdownMenu.Item key={socialItem.url} className="outline-none">
+                <a
+                  href={socialItem.url}
+                  target="_blank"
+                  rel="noreferrer"
+                  className="text-[#E0DEF2] flex text-sm px-4 py-2 hover:bg-[#454066]"
+                >
+                  {socialItem.name}
+                </a>
+              </DropdownMenu.Item>
```

Comment:
Can we have an Icon for each link?

Reply:
done, also changed gap between dropdown items to a padding in each item, to remove the little space that had between items.